docs/site/Using-extensions.md

```diff
@@ -1,18 +0,0 @@
----
-lang: en
-title: 'Using extensions'
-keywords: LoopBack 4.0, LoopBack 4
-sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Using-extensions.html
----
-
-## Extensions
-
-### Cloud native extensions
-
-- [@loopback/extension-health](Health.md) (Experimental)
-- [@loopback/extension-metrics](Metrics.md) (Experimental)
-
-### Authentication extensions
-
-- [@loopback/authentication-passport](Authentication-passport.md)
```

Comment:
Should you move this content to `Using-components.md`?

Reply:
I was considering that. But the current `Using-components.md` itself does not have a list components.